http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -232,34 +232,23 @@ class RuntimeServiceInterp[F[_]: Parallel](
 
   override def getRuntime(userInfo: UserInfo, cloudContext: CloudContext, runtimeName: RuntimeName)(implicit
     as: Ask[F, AppContext]
-  ): F[GetRuntimeResponse] =
-    for {
-      ctx <- as.ask
-      // throw 403 if no project-level permission
-      hasProjectPermission <- authProvider.isUserProjectReader(
-        cloudContext,
-        userInfo
-      )
-      _ <- F.raiseWhen(!hasProjectPermission)(ForbiddenError(userInfo.userEmail, Some(ctx.traceId)))
+  ): F[GetRuntimeResponse] = for {
+    ctx <- as.ask
 
-      // throws 404 if not existent
```

Comment:
Would be nice to keep this comment -- `RuntimeServiceDbQueries.getRuntime` will throw 404 if the runtime doesn't exist.

Reply:
Yeah, that's an annoyance I have with this Scala style: the exceptions aren't annotated. Any `IO` can throw any Throwable at any time. At least in Java you can specify `throws IOException`, for example (although you still have unchecked exceptions so it's not perfect).

Anyway that's just my thinking about why I like to leave comments about what certain things can throw. :)

Reply:
But it isn't throwing, is it? Like these come back as return values in Eithers, like `Left(SamException...))`